pypowsybl/perunitnetwork.py

```diff
@@ -0,0 +1,636 @@
+        two_windings_transformers['rated_u1'] /= two_windings_transformers['nominal_v']
+        two_windings_transformers['rated_u2'] /= two_windings_transformers['nominal_v']
+        return two_windings_transformers.drop(['nominal_v'], axis=1)
+
+    def get_3_windings_transformers(self) -> _pd.DataFrame:
+        """ Get 3 windings transformers as a ``Pandas`` data frame.
+
+        Returns:
+            a 3 windings transformers data frame
+        """
+        three_windings_transformers = self.get_elements(_pypowsybl.ElementType.THREE_WINDINGS_TRANSFORMER)
+        three_windings_transformers['p1'] /= self.sn
+        three_windings_transformers['q1'] /= self.sn
+        three_windings_transformers['p2'] /= self.sn
+        three_windings_transformers['q2'] /= self.sn
+        three_windings_transformers['p3'] /= self.sn
+        three_windings_transformers['q3'] /= self.sn
+        three_windings_transformers = _pd.merge(three_windings_transformers, self.get_voltage_levels()['nominal_v'],
+                                                left_on='voltage_level2_id', right_index=True)
+        three_windings_transformers['i1'] /= self.sn * 10 ** 3 / (
+                self.sqrt3 * three_windings_transformers['nominal_v'])
+        three_windings_transformers['i2'] /= self.sn * 10 ** 3 / (
+                self.sqrt3 * three_windings_transformers['nominal_v'])
+        three_windings_transformers['i3'] /= self.sn * 10 ** 3 / (
+                self.sqrt3 * three_windings_transformers['nominal_v'])
+        three_windings_transformers['r1'] /= three_windings_transformers['nominal_v'] ** 2 / self.sn
+        three_windings_transformers['x1'] /= three_windings_transformers['nominal_v'] ** 2 / self.sn
+        three_windings_transformers['g1'] /= self.sn / three_windings_transformers[
+            'nominal_v'] ** 2
+        three_windings_transformers['b1'] /= self.sn / three_windings_transformers[
+            'nominal_v'] ** 2
+        three_windings_transformers['r2'] /= three_windings_transformers['nominal_v'] ** 2 / self.sn
+        three_windings_transformers['x2'] /= three_windings_transformers['nominal_v'] ** 2 / self.sn
+        three_windings_transformers['g2'] /= self.sn / three_windings_transformers[
+            'nominal_v'] ** 2
+        three_windings_transformers['b2'] /= self.sn / three_windings_transformers[
+            'nominal_v'] ** 2
+        three_windings_transformers['r3'] /= three_windings_transformers['nominal_v'] ** 2 / self.sn
+        three_windings_transformers['x3'] /= three_windings_transformers['nominal_v'] ** 2 / self.sn
+        three_windings_transformers['g3'] /= self.sn / three_windings_transformers[
+            'nominal_v'] ** 2
+        three_windings_transformers['b3'] /= self.sn / three_windings_transformers[
+            'nominal_v'] ** 2
+        three_windings_transformers['rated_u1'] /= three_windings_transformers['nominal_v']
```

Comment:
Same comment: for me ratedU1 should be with the nominal voltage of the voltage level 1, ratedU2 with nominal voltage of voltage level 2, ratedU3 with the nominal voltage of voltage level 3. RatedU0 is per-unit using ratedU0 so it is equal to 1. RatedU0 is missing in your PR indeed.